parts/k8s/kubernetesprovisionsource.sh

```diff
@@ -83,7 +85,8 @@ apt_get_install() {
 systemctl_restart() {
     retries=$1; wait_sleep=$2; timeout=$3 svcname=$4
     for i in $(seq 1 $retries); do
-        timeout $timeout systemctl daemon-reload && systemctl restart $svcname && systemctl daemon-reload
```

Comment:
1) Only call `systemctl daemon-reload` _before_ calling `systemctl restart`
2) Try `systemctl restart` even if `systemctl daemon-reload` fails, because
  - we only really want to fail the deployment if the required systemd service fails to start